mlflow/tracking/_tracking_service/client.py

```diff
@@ -278,21 +278,47 @@ def log_metric(self, run_id, key, value, timestamp=None, step=None):
                       may support larger values.
         :param timestamp: Time when this metric was calculated. Defaults to the current system time.
         :param step: Training step (iteration) at which was the metric calculated. Defaults to 0.
+        :param synchronous: [Experimental] Indicates if the metric would be logged in
```

Comment:
let's add that this is default `True` (no change from current behavior) in all of these docstrings